lvm.go

```diff
@@ -51,7 +51,7 @@ func Volume(rs io.ReadSeeker) (*types.Volume, error) {
 		v.MetadataArea = append(v.MetadataArea, m)
 	}
 
```

Comment:
How about adding an error here? (I don't get a good error message)

```suggestion
if v == nil {
  return nil, xerrors.New("some error")
}
```